backend/api/prisonApi.js

```diff
@@ -363,12 +363,6 @@ const prisonApiFactory = client => {
   const getHistoryForLocation = (context, { locationId, fromDate, toDate }) =>
     get(context, `/api/cell/${locationId}/history?fromDate=${fromDate}&toDate=${toDate}`)
 
-  const moveToCell = (context, { bookingId, internalLocationDescription, reasonCode }) =>
-    put(
-      context,
-      `/api/bookings/${bookingId}/living-unit/${internalLocationDescription}?reasonCode=${reasonCode || 'ADM'}`
-    )
-
   const moveToCellSwap = (context, { bookingId }) => put(context, `/api/bookings/${bookingId}/move-to-cell-swap`, {})
```

Comment:
Is cell swap coming in a later iteration? Or are we not providing extra info for CSWAP moves?

Reply:
Yeah going to do that a bit later because I forgot all about cell swap as I was adding /cell/make-cell-move :joy: